### PR TITLE
Development

### PR DESCRIPTION
**Pull request by rpiccio**

### CODE REVIEW

File: src/main/appsuite/i18n/en_US.po [hunk 1]
```diff
@@ -9,37 +9,112 @@ msgstr ""
...
...AMAAADjcxkDAAAAilBMVEUAAAD///8ODg4KCgrU1NQGBgbJycm/v7/39/cSEhLu7u7Pz8+mpqa0tLRaWlrh4eFAQEAcHByOjo7z8/Pi4uKamppkZGQ3Nzdzc3PZ2dnp6el+fn5ubm6bm5uFhYXBwcFISEgnJydTU1MYGBgwMDB5eXkiIiK2trY0NDRnZ2eSkpJOTk6jo6M9PT2cNlPGAAAACXBIWXMAAAsTAAALEwEAmpwYAAAAB3RJTUUH3wwWFDs3AB6ggwAAAqRJREFUWMPtmAuTojAMx/NHV5BVfHC6K4663Oq5qPf9v941ffAoCMp6gzdzGaeUtpRfkzQNEoSs+l0KE4DEz3npVBySHD3qXBzBsadnEDwFBfWEczwDh/MPcGxQNhrmtlXNGAy59IKal/lBGw4fZY6XAa45WBMHWnKIrYS+3RgKf6LJmBJPbnmtD1WV6hMcrsFXzSt1yzf+lusX4PMODqDKLDR55xKZUWQ1PBPeaQ3mMG2iCDeEWGuM9bGh8JC35Y0c1+Akh1h/ijL2E2kX5hiaBYhiG8nuYWaX10X1Aq9znCraooOcbzMfGxj+/RSvFxwnGkVaH6kPJ1UcN/rHioFdog95/IS2TyLQ17x/XNZiJ8mGQFgHK628SHV/qjsKl/S6oQTwb+CALd3EjxIGBl1w9EsYp070gW7M0sxhPzCL48ZJg527/B7HWyOHaPqon/LcRo8WR9DM0W94Raz6z2qrtuT41cQxQNww+wq7THeP8w/PNos4h0+6xv2/s6pKCXbp2+XzP3RFxkN4M8D03Llf9jYHpQco24iDpakeigdTKFbhTbjBC1QjPBWFMaaoCNLMoWdXcsQ2x8EnDsy5n6ZNX+lizcBIlhcz8Cg1V88xrQA5F9VBCs1DTkH7gjsMxEOjHAeX27wmVba3uC+uo6LX5tDVONMbLA7LovaBUeKYlzAuhYy1lqOXA17XcCylvLU+b/WKwWd/gWNmb9MSh2vbpTH/qAkg+k5q3UOWhWQuqwJY6kgJZQO/zPPH2/Kx6dXdss4yYDm94Swg67qr/VWqBb4KIMUhTXkhB9VYX6dFdRxM5cx2MWFOxzHzhcgiDcWsI53hJXmLLHjI8THfcx6oRfx+/Hflf46/nI89h4DcJ/hfqpeILed0DtJTW98lp1OR8eQP/Ggcpsl69JAAAAAASUVORK5CYII="
```
Comment (jaesga) [inline src/main/appsuite/i18n/en_US.po]: What are this strings for?

File: src/main/appsuite/i18n/es_ES.po [hunk 2]
```diff
@@ -9,37 +9,112 @@ msgstr ""
...
-msgstr "Cuenta pareada"
+msgid "Latch"
+msgstr "Latch"
 
-#: apps/com.elevenpaths/latch/settings/pane.js:22
+#: apps/com.elevenpaths/latch/settings/pane.js:14
 #: module:com.elevenpaths/latch
-msgid "Unpair"
-msgstr "Desparear"
+msgid "Latch works as a security lock that protects your Webmail through a mobile app when you are not connected."
```
Comment (jaesga) [inline src/main/appsuite/i18n/es_ES.po]: This should be in Spanish right?

File: src/main/appsuite/i18n/en_US.po (hunk 1 above)
Comment (rpiccio) [inline src/main/appsuite/i18n/en_US.po]: These are the images that link to the app stores (iOS, Android, etcetera) and as some of them contain text (iOS and Android mainly) we also need to localize them.

File: src/main/appsuite/i18n/es_ES.po (hunk 2 above)
Comment (rpiccio) [inline src/main/appsuite/i18n/es_ES.po]: The msgid is the key and the default value for the message if there is no locale.pot (en_US, es_ES) file defined for the chosen locale. As we want that when no supported locale is available the messages appear in English, we have defined the keys as the messages in English.

**Pull request merged**